modules/basegl/processors/volumeprocessing/volumecombiner.cpp

```diff
@@ -169,13 +169,13 @@ void VolumeCombiner::buildShader(const std::string& eqn) {
             ss << "    vec4 " << vol << ";\n";
             ss << "    if (all(greaterThanEqual(" << coord << ", vec3(0))) &&"
                << " all(lessThanEqual(" << coord << ", vec3(1)))) {\n";
-            ss << "        " << vol << " = getNormalizedVoxel(" << v << ", " << vp << ", " << coord
+            ss << "        " << vol << " = getVoxel(" << v << ", " << vp << ", " << coord
```

Comment:
Not sure about this, I mean getVoxel will also do "normalisation" as OpenGL does it... 
we also have getSignNormalizedVoxel. Think this should probably be exposed as an option. Or someone has to convince me which one is the "correct" one...

Reply:
Done!